incubator/open-science-ce/open-science-ce/templates/service.yaml

```diff
@@ -16,6 +16,10 @@ spec:
     port: 9619
     targetPort: htcondor-ce
     protocol: TCP
+  - name: htcondor
+    port: 9618
```

Comment:
Should this be 9619?

Reply:
nope, that's the port for the CE. The port for the batch system itself (9618) also needs to be exposed.

Reply:
Ah I see now it's separate.